ShoppingM8-be/src/main/java/edu/pw/shoppingm8/product/api/dto/ProductCreateRequestDto.java

```diff
@@ -0,0 +1,18 @@
+package edu.pw.shoppingm8.product.api.dto;
+
+import javax.validation.constraints.NotNull;
+
+import lombok.Value;
+
+@Value
+public class ProductCreateRequestDto {
+    @NotNull
+    String name;
+    @NotNull
+    String unit;
+    @NotNull
+    String category;
```

Comment:
Why not enum? DTOs handle enum mapping pretty well.

Reply:
fixed